DEVELOPMENT.md

```diff
@@ -35,45 +35,54 @@ Install these tools:
     ```bash
     kind create cluster
     ```
+   **NOTE**: The KinD cluster can also be created with tools like 
+   [Podman Desktop](https://podman-desktop.io/docs/kind/creating-a-kind-cluster).
+ 
```

Comment:
I don't think its necessary, people that wants to customize things can test their own tools, for those that are new the easy way is to use kind.

Reply:
I think that using podman desktop is pretty useful for testing purposes with local kubernetes cluster. Of course it isn't needed but it is a side note and I think it's useful to have this information somewhere (it is what I would like to read when onboarding).

I am open to change it to another section if you think it's more clear, e.g., an "Optional" section under "Requirements" at the beginning of this file.

Reply:
What about moving it to DEVELOPMENT_RESOURCES.md? I want to have this DEVELOPMENT.md guide as concise, narrow and slim as possible, to not overwhelm newcomers.